reference/dtr/2.6/cli/backup.md

```diff
@@ -17,10 +17,38 @@ docker run -i --rm docker/dtr \
 
 ### Example Usage
 ```bash 
-docker run -i --rm docker/dtr \
+docker run -i --rm --log-driver none docker/dtr:latest \
     backup --ucp-ca "$(cat ca.pem)" --existing-replica-id 5eb9459a7832 > backup.tar
 ```
 
+### Example pre-populating all required information
+```bash 
+docker run -i --rm --log-driver none docker/dtr:2.6.2 backup \
```

Comment:
@omegamormegil Note to link to logging driver documentation here: https://docs.docker.com/engine/reference/run/#logging-drivers---log-driver so users will understand the context.